src/lib/xocl-main.c

```diff
@@ -125,11 +133,52 @@ static void xocl_drv_unregister_driver(enum xocl_subdev_id id)
 	pr_info("unregistered %s subdev driver\n", drvname);
 }
 
+int xocl_subdev_register_external_driver(enum xocl_subdev_id id, struct platform_driver *drv)
```

Comment:
after you fill in the platform_driver pointer, i think you also need to call xocl_drv_register_driver for this one as well?

Reply:
Done